### DISCUSSION TITLE
What is the difference between "script" and "module" actually?

### DISCUSSION THREAD

**Discussion by ghost** [eslint/eslint#15409] [General]
In the documentation:
https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options
You say:
sourceType - set to "script" (default) or "module" if your code is in ECMAScript modules.

What is the difference between "script" and "module" actually?

**Comment by nzakas** [comment]
That’s a better question for Stack Overflow. 

See https://leanpub.com/understandinges6/read/#leanpub-auto-encapsulating-code-with-modules
